DRODLib/CharacterCommand.h

```diff
@@ -200,8 +200,10 @@ namespace ScriptFlag
 		BrainedMonster = 1,
 		BestBrainTile = 2,
 		BestBrainDirection=3,
-		Puff=4,
-		Stalwart=5
+		NearestOpenRoomEdge = 4,
```

Comment:
Just thought I'd confirm with you that it's okay that we're renumbering these.

Reply:
Currently the puff and stalwart values aren't being used, so this shouldn't cause a problem. It might be cleaner to remove them, perhaps.